packages/pubsub/system-test/pubsub.js

```diff
@@ -144,11 +139,9 @@ describe('pubsub', function() {
     it('should allow manual paging', function(done) {
       pubsub.getTopics({
         pageSize: TOPIC_NAMES.length - 1
-      }, function(err, topics, nextQuery) {
+      }, function(err, topics) {
```

Comment:
Ease of use. All of our APIs support `autoPaginate`. Users shouldn't have to know what "gax" is to be able to operate our API at a basic level.